internal/discharger/login.go

```diff
@@ -73,8 +73,10 @@ type loginRequest struct {
 func (h *handler) Login(p httprequest.Params, req *loginRequest) error {
 	// Store the requested discharge ID in a session cookie so that
 	// when the redirect comes back to login-complete we know the
-	// login was initiated in this session.
-	state, err := h.params.codec.SetCookie(p.Response, waitCookieName, "/login-complete", waitState{
+	// login was initiated in this session. In case Candid is running in a sub-path,
```

Comment:
I don't think you need to add the second sentence to this comment. It's possibly better as another comment.